common/cpp/react/renderer/components/rnscreens/RNSScreenShadowNode.h

```diff
@@ -18,8 +22,26 @@ class JSI_EXPORT RNSScreenShadowNode final : public ConcreteViewShadowNode<
                                                  RNSScreenState> {
  public:
   using ConcreteViewShadowNode::ConcreteViewShadowNode;
+  using StateData = ConcreteViewShadowNode::ConcreteStateData;
```

Comment:
This `StateData` seems to be only used on `Android`, so maybe we should wrap it with `ifdef` too?

Reply:
That's compile time utility only, it does not allocate any space / do any actions during runtime. There isn't even a warning here when building iOS (just checked it out of curiosity).

Reply:
Ok, I thought rather about the code readability in this case.